pkg/puller/puller.go

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+// Package puller provides protocol-orchestrating functionality
+// over the pullsync protocol.
```

Comment:
In `pusher.go`, you write a similar comment, but add:
` (...) makes sure that chunks meant
// to be distributed over the network are sent used using the
// pushsync protocol.`
Do you think it makes sense to add a similar explainer sentence here?